remoulade/middleware/pipelines.py

```diff
@@ -71,6 +71,9 @@ def after_process_message(self, broker, message, *, result=None, exception=None)
                 if group_info and not self._group_completed(message, group_info, broker):
                     return
 
+                if "trace_ctx" in message.options:
```

Comment:
Is it possible to use `update_options_before_create_message` I find it weird to have a mention of trace_ctx in  a middleware which have nothing to do with it. We can talk if you want.

Reply:
After talking we decided to stick with this current implementation because it seems like the only way of propagating the trace context without putting in a lot of work